scripts/bench.sh

```diff
@@ -5,12 +5,37 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
+# Optimize contract's wasm binary by crate name.
+opt_wasm () {
+  local CONTRACT_CRATE_NAME=$1
+  local CONTRACT_BIN_NAME="${CONTRACT_CRATE_NAME//-/_}.wasm"
+  local CONTRACT_OPT_BIN_NAME="${CONTRACT_CRATE_NAME//-/_}_opt.wasm"
+
+  echo
+  echo "Optimising $CONTRACT_CRATE_NAME WASM binary"
+  # https://rustwasm.github.io/book/reference/code-size.html
```

Comment:
```suggestion
  # https://rustwasm.github.io/book/reference/code-size.html#use-the-wasm-opt-tool
```
nit: link directly to wasm-opt section